expertise/create_dataset.py

```diff
@@ -300,12 +301,53 @@ def retrieve_expertise(self):
 
         return expertise
 
+    def get_papers_from_group(self, submission_groups):
+        submission_groups = self.convert_to_list(submission_groups)
+        client = self.openreview_client
+        group_a_members = []
+        for group_id in submission_groups:
+            group_a_members.extend(client.get_group(group_id).members)
+        pbar = tqdm(total=len(group_a_members), desc='submissions status...')
+        publications_by_profile_id = {}
+        all_papers = []
+
+        def get_status(profile_id):
+            pbar.update(1)
+            profile = openreview.tools.get_profile(client, profile_id)
+            if profile:
+                publications = list(openreview.tools.iterget_notes(self.openreview_client, content={'authorids': profile.id}))
+                return { 'profile_id': profile_id, 'papers': publications }
+        futures = []
+        with ThreadPoolExecutor(max_workers=self.config.get('max_workers')) as executor:
+            for profile_id in group_a_members:
+                futures.append(executor.submit(get_status, profile_id))
+        pbar.close()
+
+        for future in futures:
+            result = future.result()
+            # Convert publications to json
+            papers_json_list = []
+            for paper_note in result['papers']:
+                papers_json_list.append(paper_note.to_json())
+            publications_by_profile_id[result['profile_id']] = papers_json_list
+            all_papers = all_papers + result['papers']
+
+        # Dump publications by profile id
+        with open(self.root.joinpath('publications_by_profile_id.json'), 'w') as f:
+            json.dump(publications_by_profile_id, f, indent=2)
+        
+        return all_papers
 
     def get_submissions(self):
         invitation_ids = self.convert_to_list(self.config.get('paper_invitation', []))
         paper_id = self.config.get('paper_id')
+        submission_groups = self.convert_to_list(self.config.get('alternate_match_group', []))
```

Comment:
we need to be very careful here where either paper_invitation or alternate_match_group are posible but not both at the same time.

Reply:
Sorry I missed this comment! Do you suggest I validated this in the `get_submissions` function or should I just ensure that the API does not submit a config with both of these fields?

Reply:
I would prefer to validate this in the API request class.  But in the get_submissions implementation don't load the submissions from paper_invitation and alternate_match_group. Use one only:

if invitation_ids:
    ## load the papers from the invitations
else:
   ## load the papers from alternate_match_group